migrations/000003_create_books_table.up.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE IF NOT EXISTS books (
+    id SERIAL PRIMARY KEY,
+    title VARCHAR(255) NOT NULL,
+    isbn VARCHAR(13) UNIQUE NOT NULL,
+    author_id INT NOT NULL REFERENCES authors(id) ON DELETE CASCADE,
```

Comment:
bigint

Reply:
int limitation is 2147483647, are you sure we would have more items?

Reply:
We would have less than 100 books, should we yse smallint?